test/unicode-spec.js

```diff
@@ -72,9 +111,14 @@ describe("unicode", function() {
             'ÆÇÐØÞßæçðøþ}').toParse();
     });
 
+    it("should build Latin-1 inside \\text{}", function() {
+        expect('\\text{ÀÁÂÃÄÅÈÉÊËÌÍÎÏÑÒÓÔÕÖÙÚÛÜÝàáâãäåèéêëìíîïñòóôõöùúûüýÿ' +
+            'ÆÇÐØÞßæçðøþ}').toBuild();
+    });
+
     it("should parse Latin-1 outside \\text{}", function() {
         expect('ÀÁÂÃÄÅÈÉÊËÌÍÎÏÑÒÓÔÕÖÙÚÛÜÝàáâãäåèéêëìíîïñòóôõöùúûüýÿ' +
-            'ÇÐÞçðþ').toParse();
+            'ÐÞðþ').toParse();
```

Comment:
This is backwards incompatibility: we used to (poorly) support `çÇ` in math mode. I'm not sure what's best here. At the least, removing support would mean a new major version. But we don't have good support to begin with, so maybe that's a good thing.

An alternative would be to define a macro like `\mathcedilla` (by analogy to `\mathring`) for math-mode cedilla and then we could support it in math mode. But I couldn't find any such macro in LaTeX.  We could provide one in `katex.sty` (#793) though...

Reply:
We're dropping support for `çÇ` in math mode in #1117.  I rebase after it gets merged.